playbooks/infra/deploy-bm-hypervisor.yml

```diff
@@ -0,0 +1,227 @@
+---
+- name: Create a Kickstart-enabled ISO
+  hosts: bastion
+  gather_facts: true
+  vars:
+    iso_mount_path: "{{ dest_iso_dir }}/mount"
+    os_install_path: "{{ dest_iso_dir }}/os-install"
+    location: rdu
+    system_iso_link: "{%- if location == 'rdu' -%} {{ system_iso_rdu_link }} {%- else -%} {{ system_iso_tlv_link }} {%- endif -%}"
```

Comment:
This is more clean and readable 
`system_iso_link: "{{ system_iso_rdu_link if location == 'rdu' else system_iso_tlv_link }}"
`

Reply:
THanks +1. Fixed